bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.html

```diff
@@ -0,0 +1,190 @@
+{# License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero, call_out_compact, feature_card with context %}
+
+{% from "macros.html" import fxa_email_form, fxa_link_fragment, google_play_button with context %}
+
+{% add_lang_files "firefox/whatsnew_67" %}
+
+{% extends "firefox/base-protocol.html" %}
+
+{% block page_title %}{{ _('Get a Firefox Account – Keep your data private, safe and synced') }}{% endblock %}
+{% block page_desc %}{{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One login – Power and privacy everywhere.') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_67') }}
+{% endblock %}
+
+{% set fxa_utm_params = {
+  'campaign': 'wnp67',
+  'content': '/firefox/WNP/67/',
+  'source': 'wnp-67'
+} %}
+
+{% block body_class %}state-fxa-default{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+
+<main class="mzp-t-firefox">
+  <header class="mzp-l-content c-page-header">
+    <div class="c-page-header-inner">
+      {{ high_res_img('firefox/whatsnew_67/logo.png', {'alt': 'Firefox', 'width': '147', 'height': '55', 'class': 'c-page-header-logo-fx'}) }}
+      <h1 class="c-page-header-up-to-date"><span>{{ _('Congrats! You’re using the latest version of Firefox.') }}</span></h1>
+    </div>
+  </header>
+
+  <div class="mzp-l-content">
+    {# L10n: "More power to you" is an American English idiom wishing someone success in their endeavors, and here is also a play on words saying Firefox gives users more power and control over their web experience. A more literal alternative would be "You have more power with every update" #}
+    <h2 class="c-page-title">{{ _('More power to you with every update') }}</h2>
+    <!--
+    {% set content_blocking_image='firefox/whatsnew_67/update.svg' %}
+    {% call hero(
+      heading_level=3,
+      title=_('New to your browser'),
+      desc=_('Now you have the power to block video ads that autoplay when you open articles and links. Plus, Firefox has upped your protection against ad trackers.'),
+      class='mzp-has-image mzp-l-reverse t-wn67',
+      image_url=content_blocking_image,
+      include_highres_image=false,
+      include_cta=False
+    ) %}
+    {% endcall %}
+    -->
+
+    {% call feature_card(
+      title=_('New to your browser'),
+      image_url='firefox/whatsnew_67/update.svg',
+      include_highres_image=False,
+      aspect_ratio='mzp-l-card-feature-right-66',
```

Comment:
If this design requires non-standard layout classes, suggest not using the `mpl` namespace?

Reply:
✅